test/Dommel.Tests/LoggingTests.cs

```diff
@@ -8,6 +8,7 @@
 
 namespace Dommel.Tests
 {
+    [Collection("Use Dommel Log to check on results")]
```

Comment:
What is the use of this?

Reply:
For the tests with the same collection run in sequence.